ui/src/urn.ts

```diff
@@ -48,3 +50,20 @@ export function parseIdentitySha1(urn: string): Uint8Array {
   // Drop multihash header, keep only the payload
   return hash.slice(2);
 }
+
+// Takes a binary encoded SHA1 and encodes it into a Radicle URN --
+// it's the inverse of parseIdentitySha1.
```

Comment:
Follow-up issue: Create tests for this function that assert that it is indeed the inverse of `parseIdentitySha1`. We should use property-based testing for this like [fast-check](https://github.com/dubzzz/fast-check).

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/2001